packages/lit-query/src/queryClientHelper.ts

```diff
@@ -0,0 +1,49 @@
+import type { QueryClient } from '@tanstack/query-core'
```

Comment:
I am not super happy with this solution but I have tried to make it work with [lit - Context](https://lit.dev/docs/data/context/) but I was not able to make it work because of the way it is implemented with the lifecycle.

Reply:
I can help with the lit context and some controllers. Let me know where/how.